### DISCUSSION TITLE
The TVL of ORC on DefiLlama

### DISCUSSION THREAD

**Discussion by lusterk** [DefiLlama/DefiLlama-Adapters#5909] [General]
This suggestion is an extension of https://github.com/DefiLlama/DefiLlama-Adapters/pull/5785.

  https://etherscan.io/address/0xe17d5bd0fcccab82dd76dc8ea01f6b59ae3f4754
The above case is Allbit ETH Reserve V5, which is the source of the assets received by the temporarily moved EOA.

https://etherscan.io/token/0x662b67d00a13faf93254714dd601f5ed49ef2f51?a=0xe17d5bd0fcccab82dd76dc8ea01f6b59ae3f4754
Looking at the breakdown above, you can see there are coming in and going out from various addresses to various addresses. For reference, these are the support addresses of Allbit, which was our product in the past. If you search some of the incoming addresses, you will find Coinone, a centralized exchange. As Coinone is a well-known exchange, I think you will know it without further mention.

Please check the details below.
https://etherscan.io/token/0x662b67d00a13faf93254714dd601f5ed49ef2f51?a=0xccabdda593666064cfbd2d759a8be92e68671be
https://etherscan.io/token/0x662b67d00a13faf93254714dd601f5ed49ef2f51?a=0x284dacccf7755f703007ccddf00cf4abde4d40b9
https://etherscan.io/token/0x662b67d00a13faf93254714dd601f5ed49ef2f51?a=0x9fe650ad72ebcaba08482a64ad0291df2975ac99

I would like you to reconsider ORC adding on DefiLlama. Those amounts of ORC have not belonged to the team.

**Comment by g1nt0ki** [comment]
yes, but 0xngmi says that we are not including  AXEL tokens in their tvl: https://github.com/DefiLlama/DefiLlama-Adapters/pull/5785#issuecomment-1497434411
